### ISSUE TITLE
[Bug]: load lineitem table data of TPCH 100G error 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#6538]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):b9c225f4f44dcd62d6f577394d705b6aed55188e
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

the load can not be finished , Possible causes maybe:

2022/11/13 11:05:36.769008 +0800 INFO frontend/util.go:510 time of Exec.Run : 50m6.106508677s connectionId 1015
2022/11/13 11:05:36.769056 +0800 INFO frontend/util.go:510 time of SendResponse 358ns connectionId 1015
2022/11/13 11:05:36.769109 +0800 INFO frontend/util.go:495 query trace status {"connection_id": 1015, "statement": "load data infile /data1/sudong/mo-regression/tools/mo-tpch/data/100/lineitem.tbl into table tpch_100g.lineitem fields terminated by | lines terminated by \n", "status": "success", "session_info": "connectionId 1015"}
2022/11/13 11:05:36.769322 +0800 ERROR frontend/util.go:520 rollback the txn. connectionId 1015
github.com/matrixorigin/matrixone/pkg/frontend.logErrorf
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:520
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop
        /data1/sudong/mo-regression/matrixone/pkg/frontend/routine.go:180

**Comment by qingxinhome** [comment]
Problem handling

**Comment by aressu1985** [comment]
fixed: 68177ba7729cba0dafbb7c9796170b9f755c9af4
